### DISCUSSION TITLE
Zephyr is killing itself with Discord

### DISCUSSION THREAD

**Discussion by Kezii** [zephyrproject-rtos/zephyr#76243] [General]
There is not a public and indexed knowledge base of issues and features of Zephyr

It's all kept private in a closed platform, knowledge is passed down orally between its members, in an infinite loop as it's not indexed and searchable in an efficient manner

Googling issues about Zephyr, even trivial ones, returns absolutely zero relevant results, outside poorly documented source code

please get your shit together and switch your community to an open platform

**Comment by faxe1008** [comment]
There is the officially hosted documentation over at [https://docs.zephyrproject.org/latest/index.html](https://docs.zephyrproject.org/latest/index.html). If you have concrete issues with the documentation or the search itself, pull requests are always welcome.

Regarding the tone and wording of your request, please refer to [https://github.com/zephyrproject-rtos/zephyr/blob/main/CODE_OF_CONDUCT.md](https://github.com/zephyrproject-rtos/zephyr/blob/main/CODE_OF_CONDUCT.md). Creating issues like this is not acceptable. Therefore I will close this discussion.
